docs/source/release/v6.14.0/framework.rst

```diff
@@ -10,80 +10,134 @@ Algorithms
 
 Live Data
 ---------
 
 New features
 ############
-.. amalgamate:: Framework/LiveData/New_features
+- updated header and source files for ADARA packets from v1.5.1 to v1.10.3
 
 Python
 ------
 
 New features
 ############
-.. amalgamate:: Framework/Python/New_features
+- Makes the large offline documentation an optional rather than a mandatory install, reducing installer/download size
+  significantly.
+
+  - Improvements:
+
+    - For users who frequently access online docs or have bandwidth constraints, this saves considerable disk space
+      (potentially hundreds of MB).
+    - Those who prefer local/offline usage can still opt to install the documentation package and continue working
+      without internet access.
+    - A clear indicator has been added to the Help Window’s toolbar to show whether Mantid is displaying ``Local Docs``
+      or ``Online Docs``.
+
+  - Key benefits:
+
+    - Greater flexibility in how Mantid is set up — you choose whether to save space or have full local, offline docs.
+- Introduced a prototype "side-by-side" help system that includes both the legacy QtHelp-based viewer and a new
+  Python-based Help Window using an embedded web browser (QWebEngine) to display documentation within Mantid Workbench.
+
+  - Improvements:
+
+    - Enhances the visual appearance and usability of in-app documentation.
+    - Supports richer HTML content and modern formatting, including MathJax for rendering mathematical equations.
+    - Delivers a smoother and more consistent experience when navigating help and reference material.
+
+  - Key benefits:
+
+    - Improved clarity for technical content (e.g. math and tables), more attractive and readable pages, and future
+      potential for interactive elements in documentation.
+- macOS users with Apple silicon (Arm-based architecture) are now warned if they have installed the Intel-based Mantid package.
+- Added new python algorithm, :ref:`algm-TimeDifference-v1`, for calculating the time difference between a series of runs and a reference run.
 
 Bugfixes
 ############
-.. amalgamate:: Framework/Python/Bugfixes
+
 
 
 Dependencies
 ------------------
 
 New features
 ############
-.. amalgamate:: Framework/Dependencies/New_features
+- Removed the experimental multiprocess loader from :ref:`algm-LoadEventNexus` and deprecated the ``LoadType`` property.
+- This release has removed all reliance on the `NeXus API <https://github.com/nexusformat/code>`_ .  Instead the ``File`` class was rewritten to use direct calls to the `HDF5 API <https://github.com/HDFGroup/hdf5/tree/4f1c3b6a4c7f2af6b617aede8dfb0ff1a6c58850>`_. In some places the C++ API (``H5Cpp``) is used for compatibility with other Mantid packages that also use the C++ API, but otherwise the low-level C API (``hdf5``) is used.  This decision was made because:
```

Comment:
Should this really be a release note - it does say later
`This should not impact the observed behaviour, except that memory should be more cleanly freed in Workbench, and loading or saving operations might run slightly faster.`
But maybe that's all that is required  for the user?